[2023-11-12T22:05:52.631+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509
[2023-11-12T22:05:52.633+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25'
[2023-11-12T22:05:52.690+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509
[2023-11-12T22:05:52.693+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25'
[2023-11-12T22:05:52.759+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simulator_dag_25.setup_and_run_simulator_task_25 scheduled__2023-11-11T00:00:00+00:00 [queued]>
[2023-11-12T22:05:52.782+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simulator_dag_25.setup_and_run_simulator_task_25 scheduled__2023-11-11T00:00:00+00:00 [queued]>
[2023-11-12T22:05:52.783+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-11-12T22:05:52.954+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): setup_and_run_simulator_task_25> on 2023-11-11 00:00:00+00:00
[2023-11-12T22:05:52.965+0000] {standard_task_runner.py:57} INFO - Started process 213 to run task
[2023-11-12T22:05:52.969+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simulator_dag_25', 'setup_and_run_simulator_task_25', 'scheduled__2023-11-11T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/timeseries.py', '--cfg-path', '/tmp/tmpalyh_cfu']
[2023-11-12T22:05:52.975+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask setup_and_run_simulator_task_25
[2023-11-12T22:05:53.081+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509
[2023-11-12T22:05:53.082+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_25/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_25'
[2023-11-12T22:05:53.088+0000] {task_command.py:416} INFO - Running <TaskInstance: simulator_dag_25.setup_and_run_simulator_task_25 scheduled__2023-11-11T00:00:00+00:00 [running]> on host 84c765240aba
[2023-11-12T22:05:53.208+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simulator_dag_25' AIRFLOW_CTX_TASK_ID='setup_and_run_simulator_task_25' AIRFLOW_CTX_EXECUTION_DATE='2023-11-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-11T00:00:00+00:00'
[2023-11-12T22:05:56.529+0000] {logging_mixin.py:154} INFO - Standard Output: 
[2023-11-12T22:05:56.530+0000] {logging_mixin.py:154} INFO - Standard Error: Traceback (most recent call last):
  File "/djangoapp/Task1/main.py", line 71, in <module>
    main()
  File "/djangoapp/Task1/main.py", line 21, in main
    with open("./Task1/system_config.yaml", "r") as config_file:
FileNotFoundError: [Errno 2] No such file or directory: './Task1/system_config.yaml'
[2023-11-12T22:05:56.535+0000] {logging_mixin.py:154} INFO - this is hiiiim <Thread(Thread-1, initial)>
[2023-11-12T22:05:56.537+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-12T22:05:56.550+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simulator_dag_25, task_id=setup_and_run_simulator_task_25, execution_date=20231111T000000, start_date=20231112T220552, end_date=20231112T220556
[2023-11-12T22:05:56.611+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-12T22:05:56.628+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
